### ISSUE TITLE
Features extraction at frame level 

### ISSUE THREAD

**Issue by pinkfloyd06** [shuangshuangguo/caffe2pytorch-tsn#8]
Hello @shuangshuangguo 

Let me first thank you for the conversion.

Can l extract features representation (from last fully connected layer) for each frame (rather than for the whole video ) for the RGB stream and optical flow stream ?

Thank you a lot

**Comment by shuangshuangguo** [comment]
Yes, you can do that absolutely by modifying code. Specifically, you should modify dataset.py to input one frame each time.

**Comment by Benjiou** [comment]
Thank you for your answer @shuangshuangguo, but l can't find  **dataset.py** in https://github.com/shuangshuangguo/caffe2pytorch-tsn

Thank you

**Comment by shuangshuangguo** [comment]
This repo just converse caffe model to pytorch model, thus does not contain the TSN source code. You can find dataset.py in https://github.com/yjxiong/tsn-pytorch.

**Comment by 770258506** [comment]
@shuangshuangguo Thank you for your answer，where can i find the code to extract features?
